streamdeck_ui/model.py

```diff
@@ -32,6 +34,11 @@ class ButtonState:
     """Font size of the text on the button"""
     background_color: str = ""
     """Background color of the button"""
+    plugin: Plugin = None
+    """The actual instance of the plugin, doesnt get saved in the .json"""
+    plugin_path: str = ""
+    """Path to the plugin file"""
+    plugin_config: str = ""
```

Comment:
I assume that any settings custom settings are going to be stored here. missing string doc